genienlp/ned/scripts/analyze_bootleg_results.py

```diff
@@ -0,0 +1,97 @@
+import argparse
+from collections import Counter
+
+import jsonlines
+from termcolor import colored
+
+from genienlp.ned.bootleg import BatchBootlegEntityDisambiguator
+
+parser = argparse.ArgumentParser()
+
+parser.add_argument('--input_file', type=str)
+parser.add_argument('--database_dir', type=str)
+parser.add_argument('--ned_domains', type=str, nargs='+')
+parser.add_argument('--subsample', type=int, default='1000000000')
+
+
+args = parser.parse_args()
+
+if __name__ == '__main__':
+
+    args.bootleg_model = 'bootleg_uncased_mini'
+
+    # ned_normalize_types = 'yes'
+    # args.ned_normalize_types = 'yes'
+    # args.bootleg_prob_threshold = 0.3
+    # args.max_types_per_qid = 1
+    # args.max_qids_per_entity = 1
+
+    ned_normalize_types = 'yes'
+    args.ned_normalize_types = ned_normalize_types
+    args.bootleg_prob_threshold = 0.01
+    args.max_types_per_qid = 2
+    args.max_qids_per_entity = 2
+
+    args.bootleg_output_dir = 'results_temp'
+    args.embeddings = '.embeddings'
+    args.almond_type_mapping_path = None
+
+    args.max_features_size = args.max_types_per_qid * args.max_qids_per_entity
+
+    bootleg = BatchBootlegEntityDisambiguator(args)
+
+    all_typeqids = []
+    all_aliases = []
+    all_qids = []
+    all_probs = []
+    unknown_qids = set()
+
+    lines = jsonlines.open(args.input_file, 'r')
+
+    all_titles = Counter()
+    all_new_titles = Counter()
+
+    fout = open('results.txt', 'w')
```

Comment:
This should be a command-line argument. You cannot assume you can write to the current directory.